.travis.yml

```diff
@@ -3,9 +3,12 @@ language: node_js
 node_js:
   - "6"
 
+before_install:
+  - npm install yarn -g
```

Comment:
This is currently the suggested way to get yarn on travis https://github.com/travis-ci/travis-ci/issues/6720#issuecomment-254988120

That mixed with some official support for yarn caching should make our CI super fast https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn